invenio_jobs/config.py

```diff
@@ -24,7 +24,7 @@
 JOBS_RUNS_PERMISSION_POLICY = RunPermissionPolicy
 """Permission policy for job runs."""
 
-JOBS_ADMINISTRATION_DISABLED = False
+JOBS_ADMINISTRATION_DISABLED = True
```

Comment:
We have 20+ `_ENABLED` in Invenio modules, and [only one other `_DISABLED`](https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/config.py#L295-L296).
I think we should rename it.

Reply:
I agree, changed it